msfits/MSFits/FitsIDItoMS.cc

```diff
@@ -2249,6 +2252,11 @@ void FITSIDItoMS1::fillMSMainTable(const String& MSFileName, Int& nField, Int& n
 	  gamma = 1.0;
 	}
 
+	if (itsVanVleck != 0.0) {
```

Comment:
Isn't exact comparison with a floating point number a bit scary?

Reply:
Comparison against 0.0 is perfectly safe as zero can be represented exactly. And since the memo referenced in this PR documents non-zero values as doing something different, this is exactly what is needed.

The habit if representing what should clearly be integer parameters as floats in FITS-related things is weird but that's not something we can change.